labelme/app.py

```diff
@@ -794,7 +796,6 @@ def loadFile(self, filename=None):
             self.addRecentFile(self.filename)
             self.toggleActions(True)
             self.status("Loaded %s" % os.path.basename(str(filename)))
-            self.setWindowTitle('{} - {}'.format(__appname__, os.path.basename(str(filename))))
```

Comment:
Why this change is necessary?

Reply:
Since I was added a `setWindowTitle` call to the `setDirty` method, I thought it would be clearer if this specific `setWindowTitle` call moved along with it from `loadFile` to `setClean`.

At this moment it won't make a difference if the call is in `loadFile` or in `setClean`, but if at some point an undo function would be implemented if should be in `setClean` anyway to remove the asterisk from the window title.

Reply:
Ok I see.